services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/profiili/ProfiiliController.kt

```diff
@@ -0,0 +1,51 @@
+package fi.hel.haitaton.hanke.profiili
+
+import fi.hel.haitaton.hanke.HankeError
+import io.sentry.Sentry
+import io.swagger.v3.oas.annotations.Hidden
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import mu.KotlinLogging
+import org.springframework.http.HttpStatus
+import org.springframework.security.core.annotation.CurrentSecurityContext
+import org.springframework.security.core.context.SecurityContext
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+private val logger = KotlinLogging.logger {}
+
+@RestController
+@RequestMapping("/profiili")
+class ProfiiliController(private val profiiliClient: ProfiiliClient) {
+
+    @GetMapping("/verified-name")
+    @Operation(summary = "Get verified name from Profiili")
+    @ApiResponse(description = "Success", responseCode = "200")
+    @ApiResponse(
+        description = "Verification not found.",
+        responseCode = "404",
+        content = [Content(schema = Schema(implementation = HankeError::class))]
+    )
+    @SecurityRequirement(name = "bearerAuth")
```

Comment:
This could be moved to the class in case we add other endpoints here.

Reply:
Good point, moved `@SecurityRequirement` to class level.